### PR TITLE
Have local merge models support references

### PR DESCRIPTION
**Pull request by tgsoverly**
Local merge models did not properly support `ref` functions.  I needed to push the dag down further and base more things on it.  I think this is a good thing.

### CODE REVIEW

File: test-project/local.db
Comment (jpjsonic) [inline test-project/local.db:1]: Ah my bad, I think I accidentally committed this.

File: visivo/query/jobs/run_trace_job.py [hunk 1]
```diff
@@ -55,8 +57,10 @@ def _get_target(trace, dag, output_dir):
         return targets[0]
 
     model = ParentModel.all_descendants_of_type(type=Model, dag=dag, from_node=trace)[0]
-    if isinstance(model, CsvScriptModel) or isinstance(model, LocalMergeModel):
+    if isinstance(model, CsvScriptModel):
         return model.get_sqlite_target(output_dir)
+    elif isinstance(model, LocalMergeModel):
+        return model.get_sqlite_target(output_dir, dag)
```
Comment (jpjsonic) [inline visivo/query/jobs/run_trace_job.py:63]: Is the idea that passing the dag enables nesting the local merge models?

**Review by jpjsonic — APPROVED** [review]
🚀

File: visivo/query/jobs/run_trace_job.py (hunk 1 above)
Comment (tgsoverly) [reply visivo/query/jobs/run_trace_job.py:63]: Yea because determining the models is dependent on ref, which needs the dag.

**Pull request merged**